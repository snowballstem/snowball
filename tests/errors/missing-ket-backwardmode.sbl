externals (stem)
routines (f)
backwardmode ( define f as true
