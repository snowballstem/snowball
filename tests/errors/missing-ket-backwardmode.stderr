errors/missing-ket-backwardmode.sbl:4: ) omitted after true
